app/src/main/java/org/simple/clinic/patient/PatientModule.kt

```diff
@@ -27,7 +27,9 @@ class PatientModule {
         BusinessId.MetaDataVersion.BpPassportMetaDataV1 to
             moshi.adapter(BusinessIdMetaData.BpPassportMetaDataV1::class.java) as JsonAdapter<BusinessIdMetaData>,
         BusinessId.MetaDataVersion.BangladeshNationalIdMetaDataV1 to
-            moshi.adapter(BusinessIdMetaData.BangladeshNationalIdMetaDataV1::class.java) as JsonAdapter<BusinessIdMetaData>
+            moshi.adapter(BusinessIdMetaData.BangladeshNationalIdMetaDataV1::class.java) as JsonAdapter<BusinessIdMetaData>,
+        BusinessId.MetaDataVersion.MedicalRecordNumberMetaDataV1 to
+            moshi.adapter(BusinessIdMetaData.MedicalRecordNumberMetaDataV1::class.java) as JsonAdapter<BusinessIdMetaData>
```

Comment:
Can we squash this change with the previous commit? Otherwise, that commit is broken since the metadata-adapter does not include this.